docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1272,13 +1272,21 @@
 
                 <tr>
                     <td class="property" id="store-changelog-replication-factor">stores.<span class="store">store-name</span>.changelog.<br>replication.factor</td>
-                    <td class="default">2</td>
+                    <td class="default">stores.changelog.replication.factor.default</td>
```

Comment:
prefer to change it to:
stores.changelog.default.replication.factor
It would be easier to add more default configurations for the stores.changelog configuration later.